app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,7 @@
     <string name="max_refresh_rate">Use Maximum Display Refresh Rate</string>
     <string name="max_refresh_rate_enabled">Sets the display refresh rate as high as possible (Will break most games)</string>
     <string name="max_refresh_rate_disabled">Sets the display refresh rate to 60Hz</string>
+    <string name="aspect_ratio">Choose an aspect ratio</string>
```

Comment:
We don't use this sort of phrasing for preferences, just `Aspect Ratio` is enough.